libs/input-mapping/tests/File/Options/RewrittenInputFileOptionsTest.php

```diff
@@ -178,5 +178,29 @@ public function provideFileListOptionsValues(): Generator
             'expectedOptions' => (new ListFilesOptions())->setTags(['1234-foo'])->setSinceId('1234567'),
             'originalFileOptions' => ['tags' => ['foo']],
         ];
+        yield 'single file in file_ids' => [
+            'rewrittenFileOptions' => ['file_ids' => [123]],
+            'isDevBranch' => false,
+            'runId' => '1234',
+            'filesStates' => new InputFileStateList([]),
+            'expectedOptions' => (new ListFilesOptions())->setQuery('id:(123)')->setLimit(100),
+            'originalFileOptions' => null,
+        ];
+        yield 'single file in file_ids + branch' => [
+            'rewrittenFileOptions' => ['file_ids' => [123]],
+            'isDevBranch' => false,
```

Comment:
nechtěls tady dát true? zdá se mě, že je to stejný jako předchozí case

Reply:
copy paste hell, dobrej záchyt, upravil jsem to